api/v1alpha1/grafanaorganization_types.go

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright 2024.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// GrafanaOrganizationSpec defines the desired state of GrafanaOrganization
+type GrafanaOrganizationSpec struct {
+	// DisplayName is the displayed name of the grafanaorganization. It can be different from the actual org's name.
+	DisplayName string `json:"displayName"`
+
+	// Access role that can be assumed to access the grafanaorganization.
+	RBAC *RBAC `json:"rbac,omitempty"`
+}
+
+// RBAC defines the RBAC configuration for the grafanaorganization.
+type RBAC struct {
+	// Admins is a list of user organizations that have admin access to the grafanaorganization.
```

Comment:
This is something I'm not sure of. Are Admins, Editors and Viewers **user organizations** or **users** ? From what I had understood they were supposed to be reffering to the customers' user groups.

Reply:
What's your opinion on this @QuentinBisson ?

Reply:
They are user organizations, not users

Reply:
It's up to them, it's whatever they define

Reply:
Could be teams, project, any value that comes from their ad that can differentiate tenants

Reply:
What is the difference between a user and a "user organizations" ?

Reply:
We called it org_attribute_path value here https://github.com/giantswarm/roadmap/issues/3693#issuecomment-2385805325 because we never found a name

Reply:
user is quentin@giantswarm.io, organization is atlas, platform, solution architect? That's user definable

Reply:
What are the expected values in the field then ?

Reply:
As i said, it's up to the customers so we don't know. It's a value that comes from their OIDC so it could be the user's roles, the user's groups,  a custom field. You can find more infor here https://grafana.com/docs/grafana/latest/setup-grafana/configure-security/configure-authentication/generic-oauth/#org-roles-mapping-example

Reply:
Then maybe is should be called user organization

Reply:
IIUC the list of values here would be matched against the roles defined by the auth provider at `.info.roles` correct ?

Reply:
Well no that's an example, it can map to any field within the oidc provider, here they only talk about role

Reply:
I'm not sure how to call it, oidc mapping?

Reply:
Maybe we can decide that in a quick mob session to be done with it :)

Reply:
I am having a really hard time trying to understand how this field works.

Reply:
See https://github.com/giantswarm/observability-operator/pull/116/files#r1799445250